dencam/example_config.yaml

```diff
@@ -17,3 +17,4 @@ FRAME_RATE: 25
 
 # 0, 90, 180, 270. have not checked whether 90 and 270 crop image or what
 CAMERA_ROTATION: 180 
+
```

Comment:
It would be nice if this file could not change at all i.e. it would be good to get rid of the addition of this extra line